src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java

```diff
@@ -116,7 +116,6 @@ public class CaseData implements MappableObject {
     private final OrganisationPolicy applicant1OrganisationPolicy;
     private final OrganisationPolicy respondent1OrganisationPolicy;
     private final OrganisationPolicy respondent2OrganisationPolicy;
-    private final YesOrNo respondent2SameLegalRepresentative;
```

Comment:
why are you removing this field? have you checked if this field is been used by other part?

Reply:
Good spot - I am fixing it.

Reply:
Added back - build in green.